setup.py

```diff
@@ -39,8 +39,12 @@
 if 'HPY_DEBUG' in os.environ:
     # -fkeep-inline-functions is needed to make sure that the stubs for HPy_*
     # functions are available to call inside GDB
-    EXTRA_COMPILE_ARGS = ['-g', '-O0', '-UNDEBUG',
-                          '-fkeep-inline-functions']
+    EXTRA_COMPILE_ARGS = [
+        '-g', '-O0', '-UNDEBUG',
+        '-fkeep-inline-functions',
+        #'-Wfatal-errors',    # stop after one error (unrelated to warnings)
+        #'-Werror',           # turn warnings into errors
```

Comment:
Was the commented out code left in unintentionally? Or do we want them to be on by default when `HPY_DEBUG` is set? Or something else? They do seem useful.

Reply:
They are useufl -- and that's why I left them as comments -- but they don't generally work on all systems and/or compilers.
See 943a253 and 5d2c566

Reply:
I'm not a fan of commented out code, and especially when there isn't a comment saying why the code is there. My votes are either to add an explanation in a comment (+0.5) or to remove them (+0.5) or to add another flag (`HPY_DEBUG_FAILFAST`?) that activates them.

Reply:
I think the comment is enough, no need to overengineer it. Added it in 01eaf4a

Reply:
Thanks!